Exercise_D and E/index.html

```diff
@@ -17,6 +18,62 @@
   <body>
     <!-- CODE HERE -->
 
```

Comment:
Great work here I like how you have structured your page using these HTML elements!
Superb work! Nice use of comments.